### DISCUSSION TITLE
Twitch API - need help

### DISCUSSION THREAD

**Discussion by reVolt1984** [jef/streetmerchant#1444] [General]
Hello,

I just tested the twitch integration but I'm stuck as there are no messages in the text chat of my channel after an alert via telegram and the text output was sent. Is there any way to debug the connection to twitch or at least any documentation which could help me out?

Thank you in advance.

Regards,
reVolt

**Comment by avesanidennis** [comment]
try to do the notification test, with docker:
```
docker run -it --rm  \
  --env-file ./dotenv \
  ghcr.io/jef/streetmerchant:nightly test:notification:production
```
  
without docker:
`npm run test:notification`
